framework/aster-frame/src/bus/pci/capability/msix.rs

```diff
@@ -90,6 +92,8 @@ impl CapabilityMsixData {
 
         // Set message address 0xFEE0_0000
         for i in 0..table_size {
+            #[cfg(feature = "intel_tdx")]
+            set_shared((table_bar.io_mem().paddr()) as u64, 1);
```

Comment:
I think the MSI-X table will not exceed one page size, because the size of an MSI-X entry is 16 bytes, and 256 entries are required to fill a page, which is just equal to the number of all the interrupt numbers on the x86 platform.

One device should not use so many interrupt numbers. Of course, it is better to add a judgment here in case the device deliberately uses so many interrupt numbers. In addition, due to granularity, the minimum value that can be set here is only one page.